app/src/androidTest/java/com/android/sample/screen/RecipeOverviewTest.kt

```diff
@@ -147,7 +147,8 @@ class RecipeOverviewTest {
     composeTestRule
         .onNodeWithTag(SLIDING_BUTTON_INSTRUCTIONS, useUnmergedTree = true)
         .performClick()
-    composeTestRule.onNodeWithTag(INSTRUCTIONS_VIEW, useUnmergedTree = true).assertIsDisplayed()
+    // TODO enable this
+    // composeTestRule.onNodeWithTag(INSTRUCTIONS_VIEW, useUnmergedTree = true).assertIsDisplayed()
```

Comment:
There is a Todo left here

Reply:
this doesnt come from me, from @akshire maybe ? So i will not delete it as im not the author of it